src/ui/utils/apolloClient.ts

```diff
@@ -2,10 +2,14 @@ import { ApolloClient, InMemoryCache, NormalizedCacheObject } from "@apollo/clie
 import { HttpLink } from "apollo-link-http";
 import { DocumentNode } from "graphql";
 import { assert } from "protocol/utils";
+import { getTest } from "ui/utils/environment";
```

Comment:
The intention was not to disable some graphql queries in tests but to to always seek to the endpoint in tests to make them more predictable.